lang/strings_german.txt

```diff
@@ -1010,6 +1010,8 @@ $s_custom_field = 'Feld';
 $s_custom_field_label = 'Feld:';
 $s_custom_fields_setup = 'Benutzerdefinierte Felder';
 $s_custom_field_name = 'Name';
+$s_custom_field_color_foreground = 'Vordergrundfarbe';
+$s_custom_field_color_background = 'Hintergrundfarbe';
```

Comment:
Don't change any other language file than English in code repository.

Translations are managed via TranslateWiki, which means that TranslateWiki checks about every two weeks for changes in English files.
New messages will appear on TranslateWiki as not translated, giving Mantis users the chance to translate them.
The translated messages are commited to our repository by TranslateWiki.

Please refer our own [Wiki](http://www.mantisbt.org/wiki/doku.php/mantisbt:translationshowto) for the process to submit new translations or revise existing ones.

Reply:
German language file reverted.
Didn't know that. Thanks for explanation.